include/wx/gtk/webview_webkit.h

```diff
@@ -114,7 +114,7 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewWebKit : public wxWebView
     virtual wxString GetSelectedSource() const wxOVERRIDE;
     virtual void ClearSelection() wxOVERRIDE;
 
-    virtual void RunScript(const wxString& javascript) wxOVERRIDE;
+    virtual bool RunScript(const wxString& javascript, wxString* output = NULL) wxOVERRIDE;
```

Comment:
We still want to be able to call RunScript on the WebViewWebKit1 backend, even if it has some limitations, i.e. always returns true and doesn't write the output.